python/ray/worker.py

```diff
@@ -1417,6 +1417,7 @@ def wait(object_refs, *, num_returns=1, timeout=None):
         num_returns (int): The number of object refs that should be returned.
         timeout (float): The maximum amount of time in seconds to wait before
             returning.
+        fetch_local (bool): Fetch the object to local node if it's not.
```

Comment:
Can you expand on this comment? I think it's clearer to say that if this is true, the call will only return a ref as ready once the object is fetched to the local node.